src/crimson/common/config_proxy.h

```diff
@@ -152,9 +155,19 @@ class ConfigProxy : public seastar::peering_sharded_service<ConfigProxy>
       }
     });
   }
+
   int get_val(std::string_view key, std::string *val) const {
     return get_config().get_val(*values, key, val);
   }
+
+  void set_val_default(const std::string& key, const std::string& val) {
+    get_config().set_val_default(*values, obs_mgr, key, val);
+  }
+
+  int rm_val_default(const std::string& key) {
+    return get_config().rm_val(*values, key);
+  }
```

Comment:
See above comment, same should be applied for `rm_val_default`. (see crimson::common::ConfigProxy::rm_val)

Reply:
I have updated a comment, the function is required to apply the change on a individual shard.